src/box/lua/config/instance_config.lua

```diff
@@ -232,6 +232,13 @@ local function advertise_peer_uri_validate(data, w)
     return uri
 end
 
+local function feedback_options(data)
+    if box.internal.feedback_daemon == nil then
+        return nil
+    end
+    return data
+end
```

Comment:
I think that the schema shouldn't be different on different tarantool build types. How about handling options that aren't supported by the given build in the `validate` annotation?

Reply:
Fixed.